pkg/store/bucket.go

```diff
@@ -296,6 +296,9 @@ type BucketStore struct {
 
 	// Enables hints in the Series() response.
 	enableSeriesResponseHints bool
+
+	// respBufferPool is a buffer.Bytes pool for Series() responses.
+	respBufferPool *sync.Pool
```

Comment:
I assume we don't want to limit this, right? (max num of elements it can have)

Reply:
I don't think so - it will become large just enough depending on what/how many Series() requests are coming in, and then it will gradually shrink to zero under no load

Reply:
ack